### PR TITLE
Add responsive styles for form.

### PR DESCRIPTION
**Pull request by sewest**

### CODE REVIEW

File: bmi-calculator-old/styles/styles.css [hunk 1]
```diff
@@ -17,13 +17,13 @@
   width: 100%;
 }
 
-label span {
+/* label span {
```
Comment (fseitun) [inline bmi-calculator-old/styles/styles.css:20]: no need to comment out code, that's what git's for.

File: bmi-with-grid/scratchpad.css [hunk 2]
```diff
@@ -40,8 +40,7 @@ img {
   height: 211px;
 }
 .limits-card {
-  border: 1px solid rgb(170, 0, 0); /*
-width: 100%; */
+  border: 1px solid rgb(170, 0, 0);
```
Comment (fseitun) [inline bmi-with-grid/scratchpad.css:43]: consider [css variables](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties) for properties that may repeat such as colors (maybe this isn't the case though, lol).

If your product changes it red to a pink, you'll have to search for every red in your CSS, but if you defined it in a variable you just change it in one place.

**Review by fseitun — APPROVED** [review]
Can you deploy to vercel or any free service where I can see a live demo?

File: bmi-calculator-old/styles/styles.css (hunk 1 above)
Comment (sewest) [reply bmi-calculator-old/styles/styles.css:20]: gotcha.

File: bmi-with-grid/scratchpad.css (hunk 2 above)
Comment (sewest) [reply bmi-with-grid/scratchpad.css:43]: will do.

**Comment by sewest** [comment]
deployed to vercel: https://bmi-calculator-two-phi.vercel.app/
Attached is the figma file that came with the project from code mentor.
Part of the way through building and realized that figma can show me the
grid and code etc. 😅. I should probably read their docs soon.

**Comment by fseitun** [comment]
this is really cool! congrats!!!!!

There seems to be some bug here
image

**Comment by fseitun** [comment]
As for Figma, I always brute-forced through the app, LOL.

**Comment by sewest** [comment]
Lol. Figma seems really useful, I'm just not a designer, so I haven't
looked into it yet. I've been stumbling around the interface since that's
how they provide the design files.

It doesn't have any functionality yet. I started hashing it out with
flexbox at first and then decided to re'do it mostly with grid yesterday,
so I'm just mocking up a layout atm, If that's what you mean about a bug.
Otherwise, I'm not sure what bug you mean. Is the layout behaving strangely?

**Pull request merged**